src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishASingleFileApp.cs

```diff
@@ -142,6 +144,36 @@ public void It_generates_a_single_file_including_pdbs()
                 .OnlyHaveFiles(expectedFiles);
         }
 
+        [WindowsOnlyFact]
```

Comment:
Why is this WindowsOnly?

Reply:
The `ni.pdb` files are only generated on Windows.

Reply:
ah, of course.